### PR TITLE
Implement accessibilityState

### LINKED ISSUE #4042: Upgrade to 61:  View.accessibilityState API change
In the upgrade to react-native 0.61, View got some new accessibility props:
accessibilityState, which will replace accessibilityStates.
accessibilityActions
onAccessibilityAction

For accessibilityState, see:
https://github.com/facebook/react-native/pull/24608

We'll need to add support for these new props in react-native-windows starting with 61

### PR DESCRIPTION
**Pull request by kmelmon**
Fixes #4042 

In version 0.61, accessibilityState was introduced and accessibilityStates was marked as deprecated.
In version 0.62, accessibilityStates was removed.

This change implements accessibilityState and removes accessibilityStates.

The change is straightforward - instead of an array, the property is an object with fields.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/4617)

### CODE REVIEW

**Comment by NickGerleman** [comment]
Could we split this into two changes? We want to backport accessibilityState to 0.61, but don't want to remove accessibilityStates from that branch?

File: vnext/src/Libraries/Components/View/ReactNativeViewViewConfig.windows.js [hunk 1]
```diff
@@ -140,7 +140,6 @@ const ReactNativeViewConfig = {
     accessibilityLabel: true,
     accessibilityLiveRegion: true,
     accessibilityRole: true,
-    accessibilityStates: true, // TODO: Can be removed after next release
```
Comment (NickGerleman) [inline vnext/src/Libraries/Components/View/ReactNativeViewViewConfig.windows.js:143]: This and the comment were stock 0.62. We should probably leave this as is.

Comment (kmelmon) [reply vnext/src/Libraries/Components/View/ReactNativeViewViewConfig.windows.js:143]: This is weird - I don't see any mention of accessibilityStates in the RN docs.  Is this property not actually removed yet?

Comment (NickGerleman) [reply vnext/src/Libraries/Components/View/ReactNativeViewViewConfig.windows.js:143]: It's removed to my understanding. You could quickly search their view managers in their 0.62 branch though. My guess it that fb has a view manager internally that still allows it, or just forgot to regenerate

Comment (NickGerleman) [reply vnext/src/Libraries/Components/View/ReactNativeViewViewConfig.windows.js:143]: There were other places in the 0.62 upgrade where it seemed like support was truly removed.

Comment (kmelmon) [reply vnext/src/Libraries/Components/View/ReactNativeViewViewConfig.windows.js]: OK I undid this to prepare for backporting to 61

---
In reply to: [409173188](https://github.com/microsoft/react-native-windows/pull/4617#discussion_r409173188) [](ancestors = 409173188)

**Comment by kmelmon** [comment]
OK I brought back accessibilityStates so we can backport this to 61

---
In reply to: [614312986](https://github.com/microsoft/react-native-windows/pull/4617#issuecomment-614312986) [](ancestors = 614312986)

File: vnext/ReactUWP/Views/FrameworkElementViewManager.cpp [hunk 2]
```diff
@@ -141,12 +141,21 @@ void FrameworkElementViewManager::TransferProperties(XamlView oldView, XamlView
...
+static folly::dynamic GetAccessibilityStateProps() {
+  folly::dynamic props = folly::dynamic::object();
+
+  // TODO:  "checked" is spec'd as being either a boolean or the string "mixed".  How do we handle "mixed"?
+  props.update(folly::dynamic::object("selected", "boolean")("disabled", "boolean")("checked", "boolean")(
+      "busy", "boolean")("expanded", "boolean"));
+  return props;
+}
 folly::dynamic FrameworkElementViewManager::GetNativeProps() const {
   folly::dynamic props = Super::GetNativeProps();
```
Comment (asklar) [inline vnext/ReactUWP/Views/FrameworkElementViewManager.cpp:153]: nit add newline between methods :)

**Pull request merged**